docs/draft-release-notes.md

```diff
@@ -17,7 +17,8 @@
 - PostgreSQL: The default Postgres version is now 13.4. All Postgres versions available on AWS RDS are now available to select using the "postgres_version" property.
 - PostgreSQL: binding users are now deleted when a binding is deleted. This applies only to newly created bindings.
 - PostgreSQL: a new "deletion_protection" property allows a server to be protected from accidental deletion.
-- PostgreSQL: passwords are stored using "scram". This is enabled only when not defining a custom parameter_group_name.
+- PostgreSQL: passwords are stored using "scram" encryption. This is enabled only when not defining a custom parameter_group_name.
+- PostgreSQL: Only "instance_classes" are now exposed when provisioning or updating an instance. The previous “cores” abstraction is removed, in favor of using the underlying AWS instance class property.
```

Comment:
I am assuming we are creating a new service offering right. Otherwise this looks to me like a breaking change. I would expect at least any updates to try to change the instance class, not sure how that plays out otherwise.

Reply:
Yeah we're going to expose it as a new offering. I used the same release notes as the ones in GCP